lib/KTable/index.vue

```diff
@@ -193,6 +191,15 @@
         getAriaSort,
       } = useSorting(headers, rows, defaultSort, disableBuiltinSorting);
 
+      const { delayActive, loaderVisible, wrapperInlineStyle } = useLoading(props, {
+        wrapperRef: tableWrapper,
+        loadingDelay: 300, // delay showing loader by 300ms
+        minVisibleMs: 350, // keep loader visible for at least 350ms
+        minHeightPx: 120,
```

Comment:
Since these are the defaults and there is no way to override them from outside, I'd remove these and do `useLoading(props)`.

However, if we want to give implementing products the ability to define these values themselves then we'd want to add props to KTable for the `useLoading` configuration values then pass them in here when present.

I don't think it's _necessary_ to make it something we can tweak while we use KTable in our products, but it might be a good idea in case we find that these specific values don't jive in particular contexts (ie, long loads vs short loads).